packages/scss/src/components/_timeline.scss

```diff
@@ -98,9 +98,9 @@
 
 		.gauge, .progress {
 			position: absolute;
-			left: (_theme("spacings.smallest", true) / -2);
+			left: cals(#{_theme("spacings.smallest")} / -2);
```

Comment:
@sanlucca do you prefer a `.5 *` instead of a `/ 2` ?

Reply:
no I prefer / 2 * -1 and calc instead of cals